content/docs/command-reference/import.md

```diff
@@ -74,6 +74,24 @@ Note that import stages are considered always
 they won't be updated. Use `dvc update` to update the downloaded data artifact
 from the source repo.
 
+### Backup mode
+
+To avoid duplicating large files, `dvc import` does not push imported files to
+[remote storage](/doc/command-reference/remote) by default. This means that
```

Comment:
```suggestion
To avoid duplicating large files, imported data is not pushed to
[remote storage](/doc/command-reference/remote) by default (see `dvc push`). This means that
```